### PR TITLE
add OctoBus project seed

### PR DESCRIPTION
**Pull request by businessbean**
I have added the [helm chart](https://github.com/businessbean/helm-charts/tree/master/openstack/octobus) for Octobus and i have a few questions.

- Which requirements [do i have to maintain](https://github.com/businessbean/helm-charts/blob/master/openstack/octobus/templates/seeds.yaml#L11)
- As of now we expose no services. What does it mean for the [services section](https://github.com/businessbean/helm-charts/blob/master/openstack/octobus/templates/seeds.yaml#L18)?
- How should we define the needed initial quotas?
- Can we preconfigure the Kubernikus deployment?
- Can we preconfigure DNS zones?

### CODE REVIEW

File: openstack/octobus/templates/seeds.yaml [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+  #  description: 'SAP Converged Cloud Billing services'
+  #  endpoints:
+  #  - interface: public
+  #    region: '{{.Values.global.region}}'
+  #    url: 'https://billing.{{.Values.global.region}}.cloud.sap:64000'
+
+  domains:
+  - name: ccadmin
+    projects:
+    projects:
```
Comment (ruvr) [inline openstack/octobus/templates/seeds.yaml]: Invalid / redundant structure.

projects:
- name: a project name
  description: a descriptive text

groups:
- name: group-name
   roles:
   - project: project-name
      role: role-name

etc

**Comment by ruvr** [comment]
The complete documentation of what can be seeded can be found here: https://github.com/sapcc/kubernetes-operators/tree/master/openstack-seeder

File: openstack/octobus/templates/seeds.yaml [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+metadata:
+  name: octobus-seed
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  requires:
+    # ?? what is required ??
```
[the remark this answers is not in the document]
Comment (ruvr) [reply openstack/octobus/templates/seeds.yaml]: just the ccadmin domain so far. there is no reference to anything that limes provides..

File: openstack/octobus/templates/seeds.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (businessbean) [reply openstack/octobus/templates/seeds.yaml]: Thanks. That was a typo...

File: openstack/octobus/templates/seeds.yaml [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    projects:    
+    - name: octobus
+      description: 'data collection, monitoring, analytics, event processing and storage infrastructure https://wiki.wdf.sap.corp/wiki/x/mqygd'
+      networks:
+      - name: default
+        subnets:
+         - name: octobus_private
+           cidr: 10.180.0.0/16
+           ip_version: 4
+      # Disabled because configuring subnets (instead of the parent network) during the seeding is not yet implemented
```
Comment (ruvr) [inline openstack/octobus/templates/seeds.yaml]: You can now try with:
      routers:
      - name: default
        external_gateway_info:
          network: FloatingIP-external-ccadmin-monitoring@ccadmin-net-infra@ccadmin
          external_fixed_ips:
          - subnet: FloatingIP-sap-ccadmin-monitoring-01@ccadmin-net-infra@ccadmin
        interfaces:
        - subnet: octobus_private

**Comment by viennaa** [comment]
for the record:
dns zone creation does not work

**Pull request merged**